front/src/routes/integration/all/nuki/setup-page/SetupTab.jsx

```diff
@@ -0,0 +1,77 @@
+import { Text, Localizer } from 'preact-i18n';
+import cx from 'classnames';
+
+import { RequestStatus } from '../../../../../utils/consts';
+
+const SetupTab = ({ children, ...props }) => {
+  return (
+    <div class="card">
+      <div class="card-header">
+        <h1 class="card-title">
+          <Text id="integration.nuki.setup.title" />
+        </h1>
+      </div>
+      <div class="card-body">
+        <div
+          class={cx('dimmer', {
+            active: props.nukiConnectionStatus === RequestStatus.Getting
+          })}
+        >
+          <div class="loader" />
+          <div class="dimmer-content">
+            <p>
+              <Text id="integration.nuki.setup.nukiDescription" />
+            </p>
+            {props.nukiConnectionsStatus === RequestStatus.Error && !props.nukiConnectionError && (
```

Comment:
I've tried this case by saving a random text and I get no error message: 

![Screenshot 2025-05-08 at 09 16 03](https://github.com/user-attachments/assets/953f043d-6ba5-4e70-94c9-aa48fdd79f3f)